lib/graphql.schema.js

```diff
@@ -52,6 +55,21 @@ const _getItemsQueryFromEntity = entity => {
   };
 };
 
+const _getItemQueryByIdFromEntity = (entity) => {
+  return {
+    type: _getObjectTypeFromEntity(entity),
+    args: {
+      id: {
+        description: `Fetch the ${entity.name} by id`,
```

Comment:
Move this description (of attribute) to query description and here use 'Id of fetched entity ${entity.name}'